xtokens/src/lib.rs

```diff
@@ -466,6 +480,12 @@ pub mod module {
 				currencies.len() <= T::MaxAssetsForTransfer::get(),
 				Error::<T>::TooManyAssetsBeingSent
 			);
+			for (currency_id, _) in currencies.iter() {
+				ensure!(
+					!T::OutgoingAssetsFilter::contains(currency_id),
+					Error::<T>::AssetDisabledForOutgoingTransfers
+				);
+			}
```

Comment:
I think this loop is unneeded.

Line# 506 checks this below

Reply:
oh yeah i moved it in the next loop, but forgot to delete this. thx

Reply:
agreed